src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Colors randomColor = ColorSupplier.getRandomColor();
+        Random someRandomBall = new Random();
```

Comment:
Both these objects should be constant (for `Random` you might use `Math.random()`), additionally, `someRandomBall` is not the best name for this object as it's not really a ball, but an instance of `Random` class.

Reply:
So if i understand correctly i should add term final to code and change object name or is there something more that should be changed?

Reply:
In Java, we consider a static final immutable field, named in UPPERCASE.